packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -296,6 +304,7 @@ abstract class DatabaseClusterNew extends DatabaseClusterBase {
   protected readonly newCfnProps: CfnDBClusterProps;
   protected readonly securityGroups: ec2.ISecurityGroup[];
   protected readonly subnetGroup: ISubnetGroup;
+  protected enableIamAuthentication?: boolean;
```

Comment:
What's the point of this field? Can we get rid of it please?

Reply:
Don't we need need to add that field for the db cluster?

Reply:
I don't understand the question, sorry.

Where else are you using this field other than this class?